wallet/src/components/Transactions/index.jsx

```diff
@@ -365,7 +365,7 @@ const Transactions = () => {
                     </a>
                   </div>
                 </Row>
-                {tx.txType === '3' && tx.isOnChain > 0 && !tx.withdrawState ? (
+                {tx.txType === '2' && tx.isOnChain > 0 && !tx.withdrawState ? (
```

Comment:
Could we use constants or even methods (e.g. `tx.isWithdraw()`) instead of comparing with scalar values?